smokes/e2e/buildsnavigation.scenarios.coffee

```diff
@@ -0,0 +1,67 @@
+# coffee script
+# test goal: checks the capability to navigate in a dedicated build
+# to use previous and next link
+
+
+forcePage = require('./force.coffee')
+builderPage = require('./builder.coffee')
+
+describe('', () ->
+    force = null
+    builder = null
+
+    beforeEach(() ->
+        builder = new builderPage('runtest', 'force')
+        force =  new forcePage()
+        builder.goDefault()
+    )
+
+    describe 'previousnextlink', () ->
+        it 'should navigate in the builds history by using the previous next links', () ->
+
+            # Build #1
+            builder.go()
+            builder.goForce()
+            force.getStartButton().click()
+            browser.waitForAngular()
+            browser.sleep(4000)
```

Comment:
We need to wait the build is started in the UI.
When the force button is pushed you go to the buildrequest page, when the build is started you will automatically go to the build page.

So you can just poll until the current page is a build page to now that the build has started.